workflow/rocoto/gfs_tasks.py

```diff
@@ -672,7 +672,11 @@ def marinebmat(self):
         data = f'{ocean_hist_path}/gdas.ocean.t@Hz.inst.f009.nc'
         dep_dict = {'type': 'data', 'data': data, 'offset': f"-{timedelta_to_HMS(self._base['interval_gdas'])}"}
         deps.append(rocoto.add_dependency(dep_dict))
-        dependencies = rocoto.create_dependency(dep=deps)
+        if self.app_config.do_hybvar:
```

Comment:
When you merge in develop, note that the `do_hybvar` option can now be based on `RUN` and is now stored in `self.options['do_hybvar']`.  Thus, this `if` statement should be changed to the following once you merge develop:

```python
        if self.options['do_hybvar']:
```

Reply:
OK, thanks for the heads up @DavidHuber-NOAA .